internal/common/conversion/import_state.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource"
 )
 
+func ImportSplit3(importRaw string) (ok bool, part1, part2, part3 string) {
+	parts := strings.Split(importRaw, "/")
+	if len(parts) != 3 {
+		return false, "", "", ""
+	}
+	return true, parts[0], parts[1], parts[2]
+}
+
```

Comment:
[Curious] Is the use of `/` as a separator for import applied to all the resources? Besides this question, I think you should add additional documentation to highlight this change in the TF doc

Reply:
+1 would update the resource documentation import section.

Reply:
Good point. Updated in c866b36de8b4db820543ed6a2f791617403e361c
So far, this is only used by this resource. Added a ticket [here](https://jira.mongodb.org/browse/CLOUDP-292136) to discuss more usages

Reply:
nice. Left a small comment on the import doc but this is good for me.

Reply:
This decision does indeed bring a good conversation to have if it should be extended to all resources